crates/router/src/services/authentication.rs

```diff
@@ -598,6 +599,16 @@ where
     A: AppStateInfo + Sync,
 {
     let token = get_jwt_from_authorization_header(headers)?;
+    if let Some(token_from_cookies) = get_cookie_from_header(headers)
+        .ok()
+        .map(|cookies| cookies::parse_cookie(cookies).ok())
+        .flatten()
+    {
+        logger::info!(
+            "Cookie header and authorization header JWT comparison result: {}",
+            token == token_from_cookies
+        );
```

Comment:
Are we going to remove this log in future?

Reply:
Yess. This is just for testing.